app/src/main/java/io/halkyon/services/KubernetesClientService.java

```diff
@@ -296,15 +296,15 @@ public KubernetesClient getClientForCluster(Cluster cluster) throws ClusterConne
             return client;
         } catch (Exception ex) {
             LOG.error("Error trying to get client for cluster: '" + cluster.name + "'. Caused by: " + ex.getMessage());
-            Cluster errorCluster = Cluster.findByName(cluster.name);
-            errorCluster.status = ClusterStatus.ERROR;
-            if (ex.getCause() != null) {
-                errorCluster.errorMessage = ex.getCause().getMessage();
-            } else {
-                errorCluster.errorMessage = ex.getMessage();
-            }
-            errorCluster.persist();
-            throw new ClusterConnectException(errorCluster, ex);
+            // Cluster errorCluster = Cluster.findByName(cluster.name);
+            // errorCluster.status = ClusterStatus.ERROR;
+            // if (ex.getCause() != null) {
+            // errorCluster.errorMessage = ex.getCause().getMessage();
+            // } else {
+            // errorCluster.errorMessage = ex.getMessage();
+            // }
+            // errorCluster.persist();
```

Comment:
Is there any reason to keep these comments here?

Reply:
ah no! I will remove these lines. Thx